AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -217,12 +217,11 @@ class TestScriptCanvasTests(object):
     """
     The following tests use hydra_test_utils.py to launch the editor and validate the results.
     """
-
-    @pytest.mark.skip(reason="Test fails to find expected lines, it needs to be fixed.")
     def test_FileMenu_Default_NewAndOpen(self, request, editor, launcher_platform):
         expected_lines = [
-            "File->New action working as expected: True",
-            "File->Open action working as expected: True",
+            "Verified no tabs open: True",
+            "New tab opened successfully: True",
```

Comment:
Can these strings before the ": True" part be put in a shared variable somewhere so that they don't have to be discovered in multiple places?

Reply:
I honestly don't know why we're putting these in both the TestSuite_Periodic & inside the test itself. Why don't we put those checks explicitly in the test?

Reply:
this test follows the old launch_and_verify_result log monitoring model so the suite needs to define the expected strings.
the hydra script has to write the expected lines.
the fact that the script uses Report.info to write a bool string makes it amazing that we don't just convert this over to Report.result(tuple, bool) and go to the new editor_test.py model using EditorSharedTest or another test pattern from there. that would involve reworking the entry in TestSuite_Periodic.py but would make the test more maintainable.

Reply:
@Broganab this might be related to the work you're already doing for converting the Scripting tests over to the parallelized editor tests?

@smurly I'm with you. My question to myself is, "Do we get these tests that aren't working working and cut a general task to convert all the tests, or start the conversion by converting the broken tests while fixing then go back and convert the rest when done?"

I guess that will depend on the response from @Broganab.

Reply:
@FuzzyCarterAWS I have not attempted to fix this failing test yet, so I can pull the changes into my branch to finish up the refactor work after this goes in.

Reply:
@carlitosan, had a conversation with @Broganab . They're currently doing the work that will allow us to refactor the tests to remove this kind of notation in the test suite scripts. 

To prevent collisions of work effort, and spending too much time refactoring work that will be replaced, @Broganab will be converting all of the tests that are not marked "skip" in the suite. Once that work comes in we can then convert all of the unconverted tests to the new framework and refactor the tests that need to be refactored at that time. That refactor will remove the strings called out as duplicated above.

My advice is to resolve this issue and I will add in the "convert remaining fixed tests to new editor test model" to our automation work epic.